src/Gtk/Avalonia.Gtk/GtkPlatform.cs

```diff
@@ -115,7 +115,7 @@ public IPopupImpl CreatePopup()
 
         public IRenderer CreateRenderer(IRenderRoot root, IRenderLoop renderLoop)
```

Comment:
Actually, there's a separate `IRenderFactory` interface that `CreateRenderer` is declared on - it just happens that the platform classes implement that as well. Is that OK, or would you still prefer it to be on the `ITopLevelImpl`?